src/views/connected/admin/coordinateurs/TableauCoordinateurs.js

```diff
@@ -62,7 +62,7 @@ export default function TableauCoordinateurs() {
       dispatch(paginationActions.setPage(1));
       setPage(1);
     }
-    if (!error) {
+    if (!error || error === 'Le dossier n\'existe pas') {
```

Comment:
Pour un peu simplifier l'écriture, peut-être mettre :
```suggestion
    if (!error || error === "Le dossier n'existe pas") {
```

Reply:
exigence eslint :)